tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -248,10 +253,17 @@ function cleanup_containerd() {
 
 function reset_runtime() {
 	kubectl label node "$NODE_NAME" katacontainers.io/kata-runtime-
-	systemctl daemon-reload
-	systemctl restart "$1"
-	if [ "$1" == "crio" ] || [ "$1" == "containerd" ]; then
-		systemctl restart kubelet
+
+	if [ -x /sbin/openrc-run ]; then
+		if [ "$1" == "crio" ] || [ "$1" == "containerd" ]; then
+			service restart kubelet
```

Comment:
Here you don't want to restart only kubelet, but also the containerd engine ...

Reply:
Yes!